recipes/cpp-sort/all/conanfile.py

```diff
@@ -33,13 +34,13 @@ def _minimum_compilers_version(self):
         }
 
     def validate(self):
-        if self.settings.get_safe("compiler.cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        if self.settings.get_safe("compiler.cppstd"):
```
Because header-only returns None always.

Reply:
Oh, because of the explicit `self.info.clear()` in `package_id`?

Reply:
Yes this keeps 1.x clients going